src/main/java/core/basesyntax/strategy/impl/BalanceOperationActivity.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.OperationActivity;
+
+public class BalanceOperationActivity implements OperationActivity {
+    @Override
+    public int getOperation(int balance, int amount) {
+        return balance + amount;
+    }
+}
```

Comment:
І в хендлерах зробити так, щоб вони  просто рахували, а писали в Storage потрібний результат
```suggestion
public class BalanceOperationActivity implements OperationActivity {
    @Override
    public void handleTransaction(FruitTransaction transaction) {
        Storage.map.put(transaction.getFruit(), transaction.getQuantity());
    }
}
```
Звісно інші операції будуть трохи складніші, бо треба буде витягнути поточну кількість, обрахувати результат, врахувати можливість того, що фрукту ще немає в мапі і т.і.